<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TikzExportWindow</class>
 <widget class="QWidget" name="TikzExportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to TikZ</string>
  </property>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GlobalSettings">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Global Settings</string>
   </property>
   <widget class="QCheckBox" name="toGrid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>to grid</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="toBW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>to Black&amp;White</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="toPlain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>to Plain</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
